packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -69,12 +71,105 @@ export class AccountDeleteManager {
     this.log = Container.get(AuthLogger);
   }
 
-  async deleteFirestoreCustomer(uid: string) {
-    this.log.debug('AccountDeleteManager.deleteFirestoreCustomer', { uid });
-    const result = await this.stripeHelper?.removeFirestoreCustomer(uid);
-    if (!result?.length) {
-      this.log.error('AccountDeleteManager.deleteFirestoreCustomer', { uid });
+  /**
+   * Delete the account from the FxA database, OAuth database, pushbox database, and
+   * cancel any active subscriptions.
+   *
+   * @param accountRecord
+   */
+  public async deleteAccount(accountRecord: Account) {
+    await this.deleteSubscriptions(accountRecord);
+
+    await this.deleteAccountDb(accountRecord);
+    this.log.info('accountDeleted.byRequest', accountRecord);
+
+    await this.deleteOAuthDb(accountRecord);
+
+    await this.deletePushboxRecord(accountRecord);
+
+    // TODO: Update to send notification to all devices
```

Comment:
Should we file a jira for this?  I don't want the cloud task deletes start using this until we have it.

Reply:
Follow up: https://mozilla-hub.atlassian.net/browse/FXA-8972